### PR TITLE
Remove filter threads dropdown from dashboard

### PR DESCRIPTION
**Pull request by HSalmon**
Addresses https://github.com/loomio/loomio/issues/3557

### CODE REVIEW

File: angular/core/components/dashboard_page/dashboard_page.haml [hunk 1]
```diff
@@ -1,33 +1,6 @@
 .lmo-one-column-layout
   %main.dashboard-page.lmo-row
     %section.dashboard-page__options{aria-label: "{{ 'dashboard_page.options.aria_label' | translate }}"}
```
Comment (gdpelican) [inline angular/core/components/dashboard_page/dashboard_page.haml]: Think you can lose this section as well, and the translation that goes with it. :D

File: angular/core/components/dashboard_page/dashboard_page.haml [hunk 2]
```diff
@@ -1,33 +1,6 @@
...
-                .lmo-dropdown-menu-item-details{translate: "dashboard_page.filtering.participating_description"}
-            %li.lmo-dropdown-menu-item.filtering-option.dashboard-page__filter-muted
-              %a{href: "", ng-click: "dashboardPage.setFilter('show_muted')"}
-                .lmo-dropdown-menu-item-label
-                  %span{translate: "dashboard_page.filtering.muted", aria-hidden: "true"}>
-                  %i.fa.fa-lg.fa-check{ng-show: "dashboardPage.filter == 'show_muted'"}
-                .lmo-dropdown-menu-item-details{translate: "dashboard_page.filtering.muted_description"}
-
     %h1.lmo-h1-medium.dashboard-page__heading{translate: "dashboard_page.filtering.all", ng-show: "dashboardPage.filter == 'show_all'"}
     %h1.lmo-h1-medium.dashboard-page__heading{translate: "dashboard_page.filtering.participating", ng-show: "dashboardPage.filter == 'show_participating'"}
```
Comment (gdpelican) [inline angular/core/components/dashboard_page/dashboard_page.haml]: Would be nice to drop references to participating here as well.

**Comment by robguthrie** [comment]
looks good 🍏

**Comment by HSalmon** [comment]
Good enough to go @robguthrie?

**Pull request merged**